_field-types/supported-field-types/knn-vector.md

```diff
@@ -22,19 +22,19 @@ PUT test-index
 {
   "settings": {
     "index": {
-      "knn": true,
-      "knn.algo_param.ef_search": 100
+      "knn": true
     }
   },
   "mappings": {
     "properties": {
       "my_vector": {
         "type": "knn_vector",
         "dimension": 3,
+        "space_type": "l2",
+        "mode": "in_memory",
+        "compression_level": "2x",
         "method": {
           "name": "hnsw",
-          "space_type": "l2",
-          "engine": "lucene",
           "parameters": {
             "ef_construction": 128,
             "m": 24
```

Comment:
[See if you want to update this] : we can reduce these hyper parameter values to 100, 16.

Reply:
should I just not specify?